### ISSUE TITLE
Any plan to support Firefox quantum? 

### ISSUE THREAD

**Issue by shavounet** [leibovic/fennec_rss#17]
On https://addons.mozilla.org/en-US/android/addon/home-feeds/ the add-on is marked as incompatible with Firefox quantum.

Any plan to support it?

Thanks!

**Comment by Wurftaube** [comment]
Yes. I have also this request. For me it's the reason to stay on Firefox 56.0, because my day starts with using your add-on for reading several newspaper feeds.
I would be very grateful!
Thanks!
